ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HeadersTest.kt

```diff
@@ -15,7 +15,7 @@ class HeadersTest : ClientLoader() {
     @Test
     fun testHeadersReturnNullWhenMissing() = clientTests {
         test { client ->
-            client.get<HttpResponse>("$TEST_SERVER/headers/").let {
```

Comment:
How do these changes regarding suffices and paths relate to the original purpose of the PR?

Reply:
Sorry, it was rebase gone wrong. Pushed proper changes.